themes/inpycon2018/templates/base.html

```diff
@@ -108,5 +108,7 @@
       </div>
     </div>
   </footer>
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
```

Comment:
Are the js dependencies required ? If not don't add them since more dependencies means more load time.

Reply:
Sadly, Bootstrap's Javascript doesn't work without jQuery.

![screen shot 2018-05-27 at 9 44 43 am](https://user-images.githubusercontent.com/27001046/40582394-bb122eee-6192-11e8-832b-8c34f1b1e46f.png)

Reply:
And navbar-toggle will not work without Bootstrap JavaScript.